corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+# text[eng] = There is no "sooner or later" to fortune.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Ийгиликтин>"
+	"ийгилик" n gen @obl #1->3
+"<эрте-кечи>"
+	"эрте-кеч" adj subst px3sp nom @nsubj #2->3
+"<жок>"
+	"жок" adj @root #3->0
+		"э" cop aor p3 pl @cop #4->3
+"<.>"
+	"." sent @punct #5->3
+
+# sent_id = proverbs:39
+# author:id = Сыргабековна&Сабыровна:256
+# text = Калп ырыс кесет, кайгы өмүр кесет.
+# text[eng] = A lie cuts happiness, grief cuts life.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Калп>"
+	"калп" n nom @nsubj #1->3
+"<ырыс>"
+	"ырыс" n nom @obj #2->3
+"<кесет>"
+	"кес" v tv aor p3 sg @parataxis #3->7
+"<,>"
+	"," cm @punct #4->7
+"<кайгы>"
+	"кайгы" n acc-ind @nsubj #5->7
+"<өмүр>"
+	“өмур” n nom @obj #6->7
+"<кесет>"
+	"кес" v tv aor p3 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:40
+# author:id = Сыргабековна&Сабыровна:345
+# text = Ооруну жашырсаң, өлүм ашкере кылат.
+# text[eng] = If you hide sickness/pain, death will reveal itself.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Ооруну>"
+	"оору" n acc @obj #1->2
+"<жашырсаң>"
+	"жашыр" v tv gna_cond p2 sg @advcl #2->6
+"<,>"
+	"," cm @punct #3->6
+"<өлүм>"
+	"өлүм" n nom @nsubj #4->6
+"<ашкере>"
+	"ашкере" adj @amod #5->4
+"<кылат>"
+	"кыл" v tv aor p3 sg @root #6->0
+"<.>"
+	"." sent @punct #7->6
+
+# sent_id = proverbs:41
+# author:id = Сыргабековна&Сабыровна:66
+# text = Ачуу - душман, акыл - дос, акылыңа акыл кош.
+# text[eng] = Wrath is an enemy, wisdom(/reason) is a friend; add wisdom to your wisdom.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Ачуу>"
+	"ачуу" n nom @nsubj #1->3
+"<->"
+	"-" guio @punct #2->3
+"<душман>"
+	"душман" n nom @conj #3->8
+		"э" cop aor p3 sg @cop #4->3
+"<,>"
+	"," cm @punct #5->8
+"<акыл>"
+	"акыл" n nom @nsubj #6->8
+"<->"
+	"-" guio @punct #7->8
+"<дос>"
+	"дос" n @parataxis #8->13
+		"э" cop aor p3 sg @cop #9->8
+"<,>"
+	"," cm @punct #10->14
+"<акылыңа>"
+	"акыл" n px2sg dat @obl #11->12
+"<акыл>"
+	"акыл" n nom @nsubj #12->13
+"<кош>"
+	"кош" v tv imp p2 sg @root #13->0
+"<.>"
+	"." sent @punct #14->13
+
+# sent_id = proverbs:42
+# author:id = Сыргабековна&Сабыровна:413
+# text = Тамашада да чындыктын үлүшү бар.
+# text[eng] = Even in a joke is some element of truth.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Тамашада>"
+	"тамаша" n loc @obl #1->5
+"<да>"
+	"да" postadv @advmod #2->5
+"<чындыктын>"
+	"чындык" n gen @nmod:poss #3->4
+"<үлүшү>"
+	"үлүш" n px3sp nom @nsubj #4->5
+"<бар>"
+	"бар" adj @root #5->0
+		"э" cop aor p3 sg @cop #6->5
+"<.>"
+	"." sent @punct #7->5
+
+# sent_id = proverbs:43
+# author:id = Сыргабековна&Сабыровна:178
+# text = Жалкоо - өзүнө өзү жоо.
+# text[eng] = A lazy person is an enemy to themself.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Жалкоо>"
+	"жалкоо" n nom @nsubj #1->5
+"<->"
+	"-" guio @punct #2->5
+"<өзүнө>"
+	"өз" prn ref px3sp dat @obl #3->5
+"<өзү>"
+	"өз" prn ref px3sp nom @det #4->3
```

Comment:
This doesn't seem right, but I don't have any better ideas.  @ftyers, @IlnarSelimcan?

Reply:
Maybe both are `@obl`.  That's what I'll do.

Reply:
Done in 46c3367.